Update return value of FeaturePolicy.getAllowlistForFeature (#26005)
mdn/content: files/en-us/web/api/featurepolicy/getallowlistforfeature/index.md

### Return value

> An [allowlist](/en-US/docs/Web/HTTP/Permissions_Policy#allowlists) for the

Reviewer: But this is a bit misleading isn't it?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy#allowlists looks like it is a string, containing "one or more of the following values contained in parentheses, separated by spaces". But the return value from this method seems to be an array of strings: https://w3c.github.io/webappsec-permissions-policy/#the-policy-object.

Reviewer 2: ```suggestion
An array of strings containing the serialized allowlisted origins for the feature. If a wildcard (`*`) is used, the array will contain `*`. 
```

> specified feature.

Author: ```suggestion
```

### Return value

> An array of strings containing the serialized allowlisted origins for the feature. If a wildcard (`*`) is used, the array will contain `*`. 

Author: ```suggestion
An array of strings containing the serialized list of allowed origins for the feature. If a wildcard (`*`) is used, the array will contain `*`. 
```
